cli/azd/pkg/async/task_context.go

```diff
@@ -0,0 +1,97 @@
+//nolint:all
```

Comment:
why?

Reply:
I'm still investigating this, but the linter we're using doesn't work very well with generics at the moment and we are getting some weird errors.